en/docs/reference/connectors/kafka-connector/kafka-inbound-endpoint-config.md

```diff
@@ -33,14 +33,14 @@ The following parameters are required when configuring Kafka Inbound Endpoint.
             <td>topic.name</td>
             <td>The name of the topic.</td>
         </tr>
-        <tr>
-            <td>topic.pattern</td>
-            <td>The name pattern of the topic.</td>
-        </tr>
         <tr>
             <td>contentType</td>
             <td>The content type of the message.</td>
         </tr>
+        <tr>
+            <td>topic.partitions</td>
```

Comment:
This is an optional parameter. Shall we move it under **Optional parameters for Kafka Inbound Endpoint**

Reply:
Fixed.